File: BasicInputOutput/Java/CompoundInterest.java

import java.util.Scanner;

class CompoundInterest{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Principle Amount ");
        double Principle = sc.nextDouble();
        System.out.println("Enter the rate of interest and time");
        double rate = sc.nextDouble();
        double time = sc.nextDouble();

        System.out.println("Comppound Interst : "+(Principle*rate*time)/100);
        
    }
}

File: BasicInputOutput/C/CompoundInterest.c
#include <stdio.h>
#include <math.h>

int main() {
    double principle, rate, time;
    printf("Enter the Principal Amount: ");
    scanf("%lf", &principle);
    printf("Enter the Rate of Interest and Time: ");
    scanf("%lf %lf", &rate, &time);

    double compoundInterest = principle * pow((1 + rate / 100), time) - principle;
    printf("Compound Interest: %.2lf\n", compoundInterest);

    return 0;
}
